### DISCUSSION TITLE
咨询：请问fcitx5里的“晚风”是什么输入法

### DISCUSSION THREAD

**Discussion by imxys** [fcitx/fcitx5#779] [General]
非常抱歉我以提交issue的形式来咨询我的疑问。
我对fcitx5里面的输入法一时产生了好奇，不知道这个“晚风”是什么，切换到这个输入法以后不会用。
文档也不知道哪里有，搜索过后只看到某个软件下载站点提及了“酷极晚风”输入法（2006年），似乎是一个双拼+辅助码的方案。
虽然可能多数人都习惯了全拼，请问有人对这些输入法熟悉的人吗？
打扰了。

**Comment by wengxt** [comment]
我也不知道（笑），是祖传20年的一个码表……

也许你可以去问 yuking

**Comment by waltlawrenceLL** [comment]


**Reply by fengzhiyuzhiyu** [reply]
这东西已经成为历史的一部分，没人用了

**Comment by dgod** [comment]
作者化外愚民目前仍然活跃，晚风的官方下载网址是 http://leybyr.ysepan.com/
